input/pagecontent/index.md

```diff
@@ -46,6 +46,7 @@ This Implementation Guide has been balloted as Standard for Trial Use (STU) with
 |----|----|----|----|
 | Author | Mark A. Kramer | MITRE Corporation | mkramer@mitre.org |
 | Author | Chris Moesel | MITRE Corporation | cmoesel@mitre.org |
+| Contributor | Joe Paquette | **athena**health | |
```

Comment:
Is there an ordering scheme here?  Should there be?  Alphabetical by last name?

Reply:
Alphabetized. I wanted to boost Joe's visibility, but alphabetical order seems fair.